templates/legal/nvidia-jetson-and-ubuntu.html

```diff
@@ -0,0 +1,21 @@
+{% extends "legal/_base_legal.html" %}
+
+{% block title %}NVIDIA Jetson &amp; Ubuntu{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1WI5m3vGyoPBLnXtGy0dBEGf61_eji4lkAV3OUOaMzZ4/edit{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip is-bordered">
+  <div class="u-fixed-width">
+    <h1 class="p-heading--2">NVIDIA Jetson Orin &amp; Ubuntu</h1>
+    <p>Canonical and NVIDIA partner to bring Ubuntu to popular Orin SoCs.</p>
+    <p>To learn about how the partnership will provide optimised Ubuntu on the <strong>Orin</strong> generation of Jetson products, please check out <a href="/blog/canonical-nvidia-iot-partnership">our announcement</a>.</p>
+    <h2>NVIDIA Jetson Xavier &amp; Ubuntu</h2>
+    <p>For <strong>Xavier</strong> or earlier generations of the Jetson products, if you would like to distribute NVIDIA Jetpack with a modified version of Ubuntu, Canonical would like to enter into an agreement with you to assist you to distribute NVIDIA Jetpack for Jetson in the best possible way, with the Ubuntu trademark on your distribution.</p>
+    <p>Further details about the use of Ubuntu can be found here: <a href="legal/intellectual-property-policy">https://ubuntu.com/legal/intellectual-property-policy</a></p>
```

Comment:
Comment in the copy doc to convert this to a single link instead of using a URL as a link.
```suggestion
    <p><a href="/legal/intellectual-property-policy">Further details about the use of Ubuntu</a></p>
```